aptos-move/aptos-release-builder/src/validate.rs

```diff
@@ -242,21 +213,13 @@ impl NetworkConfig {
             .inner())
     }
 
-    pub fn vote_proposal(&self, proposal_id: u64) -> Result<()> {
+    pub async fn vote_proposal(&self, proposal_id: u64) -> Result<()> {
         let address_string = format!("{}", self.validator_account);
         let privkey_string = hex::encode(self.validator_key.to_bytes());
         let proposal_id = format!("{}", proposal_id);
-        let mut root_path = std::path::Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf();
-        root_path.pop();
-        root_path.pop();
 
         let args = vec![
-            "run",
-            "--bin",
-            "aptos",
-            "--",
-            "governance",
-            "vote",
+            "",
```

Comment:
Why do we need an empty string?

Reply:
This is for the clap parsing. Clap will default the first string to be the binary name so won't be used.